src/components/SingleArticleCard.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+import Stack from "react-bootstrap/Stack";
+import Placeholder from "react-bootstrap/Placeholder";
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getDay, getHour } from "../Utils/utils";
+
+const SingleArticleCard = () => {
+	const [singleArticle, setSingleArticle] = useState({});
+	const [loading, setLoading] = useState(true);
+	const { article_id } = useParams();
+	const { id, author, body, comment_count, created_at, title, topic, votes } =
+		singleArticle;
+
+	useEffect(() => {
+		const url = `https://caragea-nc-news-backend.herokuapp.com/api/articles/${article_id}`;
+		fetch(url)
```

Comment:
dont forget to think about your errors

Reply:
Will start implementing error handling after adding functionality for all the tasks.